test/integration/features/cmd-config.feature

```diff
@@ -174,7 +175,7 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
     | iso-url         | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso  | http://github.com/minishift/minishift-centos-iso/releases/download/v1.1.0/minishift-centos7.iso |
-    | iso-url         | file:///home/me/Downloads/my_handmade_b2d.iso                                                    | file:///home/me/Downloads/my_handmade_b2d.iso                                                   |
+    | iso-url         | file:///tmp/my_handmade_b2d.iso                                                                  | file:///tmp/my_handmade_b2d.iso                                                   |
```

Comment:
@praveenkumar The function in validations is not directly unit tested, but the function in preflight checks, `checkISOUrl` is unit tested and also has negative tests. 
It is just calling the `IsValidIsoUrl` from validations..

Reply:
@anjannath Please change as per my ```request for change```

Reply:
@amitkrout Done! Thanks..

Reply:
@praveenkumar @anjannath The reason the test is important because in the past we broke the URI format for a local ISO file and test like this can catch that issue.